website/docs/gitintegration.md

```diff
@@ -0,0 +1,66 @@
+Enable git integration to run premake on checkout.
+
+```lua
+gitintegration ("value")
+```
+
+### Parameters ###
+
+| Action          | Description                                                                      |
+|-----------------|----------------------------------------------------------------------------------|
+| Off             | Disable git integration.                                                         |
+| Always          | Run premake on checkout.                                                         |
+| OnNewFiles      | Run premake only when files are added/removed or if premake script has changed.  |
+
+### Applies To ###
+
+Global scope.
+
+### Availability ###
+
+Premake 5.0.0 beta 3 or later for Visual Studio 2019 and later.
```

Comment:
Why is this logic specific to VS2019 and later?

Reply:
Bad copy paste to get *"Premake 5.0.0 beta 3 or later"*.

Reply:
Fixed.